s4e-backend/src/main/resources/db/migration/V34__add_indexes_metadata_scene.sql

```diff
@@ -0,0 +1,14 @@
+CREATE OR REPLACE FUNCTION f_cast_isots(text)
+  RETURNS timestamptz AS
+$$SELECT to_timestamp($1, 'YYYY-MM-DD"T"HH24:MI:SS"Z"')$$
+  LANGUAGE sql IMMUTABLE;
```

Comment:
Wouldn't it be better to do that in a repeatable migration? Just wondering actually, I'm not really sure it's the way to go.